src/components/tasks/card/progressBar/index.tsx

```diff
@@ -0,0 +1,116 @@
+import { FC, useState } from 'react';
+import { useRouter } from 'next/router';
+
+import { toast, ToastTypes } from '@/helperFunctions/toast';
+import {
+    useUpdateSelfTaskMutation,
+    useUpdateTaskMutation,
+} from '@/app/services/tasksApi';
+import useUserData from '@/hooks/useUserData';
+import { useGetUserQuery } from '@/app/services/userApi';
+
+import HandleProgressText from './ProgressText';
+import HandleProgressbar from './ProgressBar';
+import { ERROR_MESSAGE, PROGRESS_SUCCESSFUL } from '@/constants/constants';
+
+import classNames from '@/components/tasks/card/card.module.scss';
+import { ProgressContainerProps } from '@/interfaces/task.type';
+
+const ProgressContainer: FC<ProgressContainerProps> = ({ content }) => {
+    const router = useRouter();
+    const { dev } = router.query;
+
+    const [progress, setProgress] = useState<boolean>(false);
```

Comment:
Since this is a boolean value, it would be better to rename it so that it's clear it will hold boolean value. Something like `hasProgressStated`, `isProgressMade` etc type of naming variables.

Reply:
fixed